lib/generateToken.js

```diff
@@ -1,28 +1,35 @@
 'use strict';
 
-const nodeify = require('./nodeify');
-const crypto = require('crypto');
 const base64url = require('base64url');
+const crypto = require('crypto');
+const nodeify = require('./nodeify');
+
+// Config for pbkdf2
 const ALGORITHM = 'sha256';
-const TOKEN_LENGTH = 256; // Measured in bits
+const ITERATIONS = 100000;
```

Comment:
Not a blocker. It may be note-worthy to state why this constant was chosen. I understand it's recorded in the PR, so that may be sufficient enough.